scripts/sv/alignAssembly.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu
+
+if [[ -z ${GATK_DIR+x} || -z ${CLUSTER_NAME+x} || -z ${MASTER_NODE+x} || -z ${PROJECT_OUTPUT_DIR+x} ]]; then 
+    if [[ "$#" -ne 3 ]]; then
+        echo "Please provide: local directory of GATK build, cluster name, output dir on the cluster"
+    exit 1
+    fi
+    GATK_DIR=$1
+    CLUSTER_NAME=$2
+    OUTPUT_DIR=$3
+    MASTER_NODE="hdfs://""$CLUSTER_NAME""-m:8020"
+    PROJECT_OUTPUT_DIR="$MASTER_NODE"/"$OUTPUT_DIR"
+fi
+
+cd "$GATK_DIR" 
+
+./gatk-launch AlignAssembledContigsSpark \
+    -R "$MASTER_NODE"/reference/Homo_sapiens_assembly19.fasta \
```

Comment:
Could you add top-level comments to scripts that are dependent on using hg19/b37 versions of the reference that highlights that fact? Just want to avoid any future pain on the part of someone who doesn't realize that. In a future PR we could figure out how to parameterize that (or hopefully just move away from needing these scripts altogether).